modules/ROOT/examples/java/src/com/couchbase/code_snippets/Examples.java

```diff
@@ -503,7 +502,7 @@ implementation fileTree(dir: 'libs', include: '*.jar')
 // end::GsWebAppBuildGradle[]
 
 // tag::GsEmbeddedTomcatBuildGradle[]
-def tomcatVersion = '9.0.24'
```

Comment:
??? Why not use the variable.  It forces consistency across versions...
Am I missing something?